index.bs

```diff
@@ -11302,6 +11306,174 @@ The [=remote end steps=] given |session| and |command parameters| are:
+
+1. Let |result| be a [=/map=] matching the
+   <code>webExtensions.InstallResult</code> production with the
+   <code>extension</code> field set to |extension id|.
+
+1. Return [=success=] with data |result|.
+
+</div>
+
+<div algorithm>
+To <dfn>expand a web extension data spec</dfn> given |extension data spec|:
+
+1. Let |type| be |extension data spec|["<code>type</code>"].
+
+1. If installing a web extension using |type| isn't supported return [=error=] with [=error code=] [=unsupported operation=].
+
+1. In the following list of conditions and associated steps, run the first set of steps for which the associated condition is true:
+
+   <dl>
+    <dt>|type| is the string "<code>path</code>"
+    <dd>
+      1. Let |path| be |extension data spec|["<code>path</code>"].
+      1. Let |entry| be a [=directory entry=] representing the directory with that |path|.
+
+    <dt>|type| is the string "<code>archivePath</code>"
+    <dd>
+      1. Let |archive path| be a value of <code>path</code> field of |extension data spec|.
+      1. Perform any implementation-defined steps to extract ZIP from |archive path| and let |entry| be a [=directory entry=] representing the extraction result, if this fails return [=error=] with [=error code=] [=invalid web extension=].
+
+    <dt>|type| is the string "<code>base64</code>"
+    <dd>
+      1. Let |bytes| be [=forgiving-base64 decode=] |extension data spec|["<code>path</code>"].
+      1. Perform any implementation-defined steps to extract the archive from |bytes| by using the ZIP algorithm, and let |entry| by a [=directory entry=] representing the extraction path, if this fails return [=error=] with [=error code=] [=invalid web extension=].
+
+   </dl>
+
+1. Return [=success=] with data |entry|.
+
+</div>
+
+#### The webExtensions.uninstall Command #### {#command-extensions-uninstall}
+
+The <dfn export for=commands>webExtensions.uninstall</dfn> command uninstalls a web extension for the session.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      webExtensions.Uninstall = (
+         method: "webExtensions.uninstall",
+         params: webExtensions.InstallParameters
+      )
+
+      webExtensions.UninstallParameters = {
+         extension: webExtensions.Extension,
+      }
+      </pre>
+   </dd>
+</dl>
+
+<div algorithm="remote end steps for webExtensions.uninstall">
+The [=remote end steps=] with |command parameters| are:
+
+1. Let |extension| be a value of <code>extension</code> field of the |command parameters|.
+
+1. If the browser has no web extension with an id equal to |extension|, return [=error=] with [=error code=] [=invalid argument=].
+
+1. Perform any implementation-defined steps to remove the web extension from the browser. If this fails, return [=error=] with [=error code=] [=unknown error=].
```

Comment:
Above we talk about the session and here the browser. I assume no browser can actually install the extension for a webdriver session only. So maybe we should re-phrase the initial description for both commands?

Reply:
WebExtension are installed in a browser profile. [It is advisable that remote ends create a new profile when creating a new session](https://w3c.github.io/webdriver/#privacy). So should we reference a profile instead of a browser or session?

Reply:
I would be fine by using `browser`.